.github/CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 ### New features since last release
 
+* Add multi-node/multi-gpu support to gate operation.
+  [(#112)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/112)
+
+  This new feature allows users to run their large-scale applications by leveraging the computational power of multi-node and multi-gpus.
+  Note that both the number of overall `MPI` processes and the number of `MPI` processes per node should be power of `2`. Each `MPI` process
```

Comment:
Why should the number of `MPI` processes per node be 2**N? Not complying will slow down certain operations, but is it a strict requirement?

Reply:
It's for performance purpose. For the moment, one MPI process is for one GPU device.  We need to make sure each nodes has 2**N MPI processes to set up P2P bits.

Reply:
That's what I'm thinking, so worst case scenario, it is slower for operations targeting precisely those qubits. So if there aren't 2**N GPU/proc per node, could we just turn off P2P bits? I'm thinking of Summit, where you have 6 GPUs per node. If you have 4 global qubits, this requirement would force a user to use 4 nodes and waste 2 GPUs per node. Otherwise, one could use 16 processes on 3 nodes and waste only 2 GPUs total.